pkg/cc/config/templates.go

```diff
@@ -144,6 +144,23 @@ api:
     console:
       gateway:
         listen_address: "0.0.0.0:8080"
+        allowed_headers:
+        - "Authorization"
+        - "Content-Type"
+        - "If-Match"
+        - "If-None-Match"
+        - "Depth"
+        allowed_methods:
+        - "GET"
+        - "POST"
+        - "HEAD"
+        - "DELETE"
+        - "PUT"
+        - "PATCH"
+        - "PROFIND"
+        - "MKCOL"
+        - "COPY"
+        - "MOVE"
```

Comment:
We don't expose any endpoints that respond to these methods, do we? Why allow them by default?